mvp/resources/README

```diff
@@ -1 +1,71 @@
-Dummy readme
\ No newline at end of file
+IBM Distribution for Galasa version 1.0 (20210618)                                                                                                                
```

Comment:
either the timestamp need to be autogenerated by the build or we need to lose it

Reply:
why do we need a timestamp?

Reply:
🤷

Reply:
will remove